src/fastq/fastqread.jl

```diff
@@ -0,0 +1,64 @@
+# FASTQ Read
+# ==========
+#
+# Design discussions in https://github.com/BioJulia/FASTX.jl/pull/35
+
+mutable struct FASTQRead
+    identifier::String
+    description::String
+    sequence::BioSequences.LongDNASeq
+    quality::Vector{UInt8} # in raw PHRED scores, not offset
+end
+
+"""
+    FASTQ.Read(record::FASTQ.Record, offset::Integer=33)
+
+Create a FASTQ read object from a FASTQ.Record.
+The FASTQ.Read has fields: 
+    identifier::String
+    description::String
+    sequence::BioSequences.LongDNASeq
+    quality::Vector{UInt8} # in raw PHRED scores, not offset
+"""
+function Read(record::FASTQ.Record, offset::Integer=33)
+    FASTQRead(
+        identifier(record),
+        description(record),
+        sequence(BioSequences.LongDNASeq, record),
```

Comment:
Also here, it should be generic for any sequence type

Reply:
fixed in f50038e